int64 i
---
int64[] j_list